streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -266,13 +272,13 @@ public void suspend() {
             case CREATED:
                 log.info("Suspended created");
                 transitionTo(State.SUSPENDED);
-
+                timeCurrentIdlingStarted = Optional.of(System.currentTimeMillis());
```

Comment:
Since this is coupled with suspend state transition, could we refactor a helper method instead?